runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -123,6 +175,35 @@ public void onVarbitChange(VarbitChanged event)
 			removeGameTimer(PRAYER_ENHANCE);
 			lastRaidVarb = raidVarb;
 		}
+
+		int inWilderness = client.getVar(Varbits.IN_WILDERNESS);
+		if (lastWildernessVarb != inWilderness)
+		{
+			if (worldType == NORMAL && inWilderness == 0)
+			{
+				removeTbTimers();
+			}
+			lastWildernessVarb = inWilderness;
+		}
+	}
+
+	@Subscribe
+	public void onWidgetHiddenChanged(WidgetHiddenChanged event)
+	{
+		if (worldType == PVP)
```

Comment:
This check is present because entering the wilderness on a normal world will trigger this condition as most of the pvp-related widgets load un-hidden and are thereafter hidden as needed. (that is, the safe zone widget is loaded un-hidden then immediately hidden before the frame renders)

With that in mind, world hopping between PVP worlds may need to be checked for as this same condition may be met there.

Reply:
After testing, my suspicion here was correct--hopping between PVP worlds in a non-safe zone with this current code will register a safe zone trigger because of this phenomenon. Unfortunately, the `GameState` is `LOGGED_IN` by the time it happens, so this can't be gated behind a check for that. Does anyone have an idea of how to tackle this issue? To my knowledge, I can't add an async check for this, otherwise I'd just add some kind of short delay (say, 100ms or less) to check the widget state at that point.